packages/itauto/itauto.8.13+no/opam

```diff
@@ -0,0 +1,34 @@
+opam-version: "2.0"
+maintainer: "frederic.besson@inria.fr"
+homepage: "https://gitlab.inria.fr/fbesson/itauto"
+dev-repo: "git://gitlab.inria.fr:fbesson/itauto.git"
+authors: ["Frédéric Besson"]
+bug-reports: "frederic.besson@inria.fr"
+license: "GPL 3"
+synopsis: "'itauto' is a reflexive SAT solver parameterised by a leaf tactic and Nelson-Oppen support"
+build: [
+  [make]
+]
+install: [
+  [make "install"]
+]
+remove: ["rm" "-R" "%{lib}%/coq/user-contrib/Cdcl"]
```

Comment:
```suggestion
```
It's not a blocker but if you're not adding/modifying any files in this directory it should be safe to not have any `remove` field as opam trasks all the files added during install.